.github/fabricbot.json

```diff
@@ -1709,7 +1709,8 @@
           ],
           "mentionees": [
             "toddysm",
-            "northtyphoon"
+            "northtyphoon",
+            "ShivangiReja"
```

Comment:
Shivangi is not part of the service team and should not be mentioned for "Service Attention" issues.

Reply:
Removed Shivangi